### PR TITLE
jenkinsfile fix

### PR DESCRIPTION
**Pull request by odra**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -66,7 +66,7 @@ node(platform) {
 
     stage("Archive") {
         if (platform == 'android') {
-            archiveArtifacts artifacts: 'platforms/android/build/outputs/apk/android-${BUILD_CONFIG}.apk', excludes: 'platforms/android/build/outputs/apk/*-unaligned.apk'
```
Comment (laurafitzgerald) [inline Jenkinsfile:69]: does the excludes value also require double quotes or is that okay?

Comment (odra) [reply Jenkinsfile:69]: it's ok as it does not use any variables, we need to use double quotes to parse a variable(s) value(s).

Comment (laurafitzgerald) [reply Jenkinsfile:69]: ok thanks for clarification. the change looks good. Is it deployed somewhere to test?

**Review by laurafitzgerald — APPROVED** [review]
.

**Pull request merged**